tests/test_device/test_alpa_dp.py

```diff
@@ -0,0 +1,15 @@
+from colossalai.device import get_submesh_choices
+
+
+def test_alpa_dp():
```

Comment:
I think you should add some assertions in unit test. I know the profiling result may have a large variance, but you could check whether the result in reasonable range, such as latency less than 100ms.